zaas-client/src/main/java/org/zowe/apiml/zaasclient/service/internal/ZaasJwtService.java

```diff
@@ -32,15 +32,15 @@
 import java.util.stream.Stream;
```

Comment:
Also possible to replace "Set-cookie" with SM.SET_COOKIE (line 118)

Reply:
Done